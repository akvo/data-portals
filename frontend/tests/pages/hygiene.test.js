import React from 'react'
import { render } from '@testing-library/react'
import Hygiene from '../../pages/hygiene'

describe('Hygiene page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Hygiene />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
