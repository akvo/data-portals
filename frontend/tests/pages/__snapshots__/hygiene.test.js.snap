// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hygiene page matches snapshot 1`] = `
<DocumentFragment>
  <h1>
    Hygiene
  </h1>
</DocumentFragment>
`;
